frontend-opal/components/Header/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+import {
+    Button,
+    View,
+    TextInput,
+    Image,
+} from 'react-native'
+import styles from './styles.js'
+
+const Header = () => {
+    const narTitles = [
+        { id: 1, key: "PORTFOLI" },
+        { id: 2, key: "WATCHLIST" }
+    ]
+
+    const handlePress = (key) => key === "PORTFOLIO";
+
+    const renderItems = (titles) => (
+        titles.map(title => (
+            <View
+                key={title["id"]}
+                style={[styles.narTitle, handlePress(title["key"]) && styles.buttonPressed]}
+            >
+                <Button
+                    title={title["key"]}
+                    color="#e17055"
+                />
+            </View>
+        ))
+    )
+
+    return (
+        <View>
+            {/* Three children can be wrapped into components */}
```

Comment:
@Yuan-EPAM don't leave magical comments in the code, refactor it if it's dangerous without the comment